pkg/keygen/keygen.go

```diff
@@ -22,23 +22,23 @@ type Generator struct {
 	Store *storage.Filesystem
 }
 
-// KeyForMetadata generates a 2048-bit RSA private key, or returns an existing one
-// if the reuse private key attribute is present.
+// KeyForMetadata generates a 2048-bit RSA private key, or returns an existing
```

Comment:
Question: any plans for allowing other types of key?

Reply:
^